### PR TITLE
Large Install Tuning - Taskomatic jobs

### PR DESCRIPTION
**Pull request by Loquacity**
"backport" of https://github.com/SUSE/doc-susemanager/pull/466

### CODE REVIEW

File: adoc/advanced_topics_optimization_scalability.adoc [hunk 1]
```diff
@@ -187,8 +187,14 @@ Keep the server monitored in order to identify possible issues early.
...
 ** Increase this parameter further if file management states fail with the error "Unable to manage file: Message timed out"
 ** Note that Salt master workers can consume significant amounts of RAM (typically about 70{nbsp}MB per worker). It is recommended to keep usage monitored when increasing this value and to do so in relatively small increments (eg. 20) until failures are no longer produced.
-* Disable daily comparison of configuration files. Click on menu:Admin[Task Schedules], then on the btn:[compare-configs-default] link, then on the btn:[Disable Schedule] button and finally on btn:[Delete Schedule].
 * Increase the maximum heap memory for the search daemon to be able to index many minions. Set 4 GiB instead of the current default 512 MB: add [parameter]``rhn-search.java.maxmemory=4096`` in [path]``/etc/rhn/rhn.conf`` (affects background indexing only).
+* Consider disabling Taskomatic jobs, especially if the provided functionality is not used:
+** Disable daily comparison of configuration files.
+Click on menu:Admin[Task Schedules], then on the btn:[compare-configs-default] link, then on the btn:[Disable Schedule] button and finally on btn:[Delete Schedule].
```
Comment (keichwa) [inline adoc/advanced_topics_optimization_scalability.adoc]:
```suggestion
Click menu:Admin[Task Schedules], then the btn:[compare-configs-default] link, then the btn:[Disable Schedule] button and finally btn:[Delete Schedule].
```

File: adoc/advanced_topics_optimization_scalability.adoc [hunk 2]
```diff
@@ -187,8 +187,14 @@ Keep the server monitored in order to identify possible issues early.
...
+** Disable hourly synchronization of Cobbler files.
+Click on menu:Admin[Task Schedules], then on the btn:[cobbler-sync-default] link, then on the btn:[Disable Schedule] button and finally on btn:[Delete Schedule].
```
Comment (keichwa) [inline adoc/advanced_topics_optimization_scalability.adoc]:
```suggestion
Click menu:Admin[Task Schedules], then the btn:[cobbler-sync-default] link, then the btn:[Disable Schedule] button and finally btn:[Delete Schedule].
```

File: adoc/advanced_topics_optimization_scalability.adoc [hunk 3]
```diff
@@ -187,8 +187,14 @@ Keep the server monitored in order to identify possible issues early.
...
+** Disable daily run of Gatherer and Subscription Matcher.
+Click on menu:Admin[Task Schedules], then on the btn:[gatherer-matcher-default] link, then on the btn:[Disable Schedule] button and finally on btn:[Delete Schedule].
```
Comment (keichwa) [inline adoc/advanced_topics_optimization_scalability.adoc]:
```suggestion
Click menu:Admin[Task Schedules], then the btn:[gatherer-matcher-default] link, then the btn:[Disable Schedule] button and finally btn:[Delete Schedule].
```

File: adoc/advanced_topics_optimization_scalability.adoc [hunk 4]
```diff
@@ -187,8 +187,14 @@ Keep the server monitored in order to identify possible issues early.
...
 
 Note that increasing the number of Postgres connections will require more RAM, make sure the {productname} server is monitored and swap is never used.
```
Comment (keichwa) [inline adoc/advanced_topics_optimization_scalability.adoc]: I think we still prefer "PostgreSQL".

**Comment by Loquacity** [comment]
Updated https://github.com/SUSE/doc-susemanager/pull/466 as per Karl's feedback.

**Pull request merged**